src/domain/app.ts

```diff
@@ -45,6 +45,6 @@ export const MempoolExplorerURLs: ExplorerURLs = {
 
 export const MempoolTestnetExplorerURLs: ExplorerURLs = {
   type: 'Mempool',
-  electrsURL: 'https://liquid.network/liquidtestnet',
+  electrsURL: 'https://liquid.network/testnet',
```

Comment:
I think is opposite. electrs is bakcend and esploraURL is frontend?

Reply:
On the other providers is not:

```typescript
export const BlockstreamExplorerURLs: ExplorerURLs = {
  type: 'Blockstream',
  electrsURL: 'https://blockstream.info/liquid',
  esploraURL: 'https://blockstream.info/liquid/api',
};

export const BlockstreamTestnetExplorerURLs: ExplorerURLs = {
  type: 'Blockstream',
  electrsURL: 'https://blockstream.info/liquidtestnet',
  esploraURL: 'https://blockstream.info/liquidtestnet/api',
};

export const NigiriDefaultExplorerURLs: ExplorerURLs = {
  type: 'Nigiri',
  electrsURL: 'http://localhost:5001',
  esploraURL: 'http://localhost:3001',
};

export const MempoolExplorerURLs: ExplorerURLs = {
  type: 'Mempool',
  electrsURL: 'https://liquid.network',
  esploraURL: 'https://liquid.network/api',
};

export const MempoolTestnetExplorerURLs: ExplorerURLs = {
  type: 'Mempool',
  electrsURL: 'https://liquid.network/testnet',
  esploraURL: 'https://liquid.network/liquidtestnet/api',
};
```